core/services/workflows/syncer/workflow_registry.go

```diff
@@ -379,30 +379,51 @@ func (w *workflowRegistry) newWorkflowRegistryContractReader(
 				Configs: map[string]*evmtypes.ChainReaderDefinition{
 					GetWorkflowMetadataListByDONMethodName: {
 						ChainSpecificName: GetWorkflowMetadataListByDONMethodName,
+						ConfidenceConfirmations: map[string]int{
+							"1.0": 3,
+						},
 					},
 					string(ForceUpdateSecretsEvent): {
 						ChainSpecificName: string(ForceUpdateSecretsEvent),
 						ReadType:          evmtypes.Event,
+						ConfidenceConfirmations: map[string]int{
+							"1.0": 3,
+						},
 					},
 					string(WorkflowActivatedEvent): {
 						ChainSpecificName: string(WorkflowActivatedEvent),
 						ReadType:          evmtypes.Event,
+						ConfidenceConfirmations: map[string]int{
+							"1.0": 3,
+						},
 					},
 					string(WorkflowDeletedEvent): {
 						ChainSpecificName: string(WorkflowDeletedEvent),
 						ReadType:          evmtypes.Event,
+						ConfidenceConfirmations: map[string]int{
+							"1.0": 3,
+						},
 					},
 					string(WorkflowPausedEvent): {
 						ChainSpecificName: string(WorkflowPausedEvent),
 						ReadType:          evmtypes.Event,
+						ConfidenceConfirmations: map[string]int{
+							"1.0": 3,
+						},
 					},
 					string(WorkflowRegisteredEvent): {
 						ChainSpecificName: string(WorkflowRegisteredEvent),
 						ReadType:          evmtypes.Event,
+						ConfidenceConfirmations: map[string]int{
+							"1.0": 3,
+						},
 					},
 					string(WorkflowUpdatedEvent): {
 						ChainSpecificName: string(WorkflowUpdatedEvent),
 						ReadType:          evmtypes.Event,
+						ConfidenceConfirmations: map[string]int{
+							"1.0": 3,
```

Comment:
Ideally, you'd set a more realistic confidence level for the chain, like "0.9":  3 and pass that instead of `Finalized` to `GetLatestValue` , `QueryKey`, etc. and reserve 1.0 for meaning actually finalized. But if you're sure that nothing else that shares this config will ever need a higher confidence level, this works.

Reply:
@EasterTheBunny would know better than I though whether there is any problem with doing it this way